File: lib/entities/Product.dart
class Product {
  final String? code;
  final String name;
  final String description;
  final String imageUrl;
  final String environmentalInfo;
  final String category;
  final String environmentalCategory;

  Product({
    this.code,
    required this.name,
    required this.description,
    required this.imageUrl,
    required this.environmentalInfo,
    required this.category,
    required this.environmentalCategory
  });
}

File: backend/src/models/Product.js
import { Schema, model} from 'mongoose'

const productSchema = new Schema ({
    code: String,
    name: String,
    description: String,
    imageUrl:String,
    environmentalInfo:String,
    category:String,
    environmentalCategory:String,
})

export default   model('Product', productSchema);
